mlflow/utils/_spark_utils.py

```diff
@@ -36,6 +38,30 @@ def _prepare_subprocess_environ_for_creating_local_spark_session():
     os.environ.pop("PYSPARK_GATEWAY_SECRET", None)
 
 
+def _get_spark_scala_version_from_spark_session(spark):
+    version = Version(spark._jvm.scala.util.Properties.versionNumberString())
+    return f"{version.major}.{version.minor}"
+
+
+def _get_spark_scala_version():
+    from pyspark.sql import SparkSession
+
+    from mlflow.utils.databricks_utils import is_in_databricks_runtime
+
+    if is_in_databricks_runtime():
+        return os.environ["SPARK_SCALA_VERSION"]
```

Comment:
```suggestion
        return "SPARK_SCALA_VERSION" in os.environ
```

Reply:
Never mind this.

Reply:
Does this environment variable always exist?

Reply:
I tested on DBR 11 ~ 15, they all have SPARK_SCALA_VERSION env variables.

For safety, I also add fallback approach for it